test/unit/util/hash-project-loader-hoc.test.jsx

```diff
@@ -0,0 +1,64 @@
+import React from 'react';
+import ProjectLoaderHOC from '../../../src/lib/project-loader-hoc.jsx';
+import HashParserHOC from '../../../src/lib/hash-parser-hoc.jsx';
+import storage from '../../../src/lib/storage';
+import {mount} from 'enzyme';
+
+jest.mock('react-ga');
+
+describe('Hash/ProjectLoaderHOC', () => {
+    test('when there is no project data, it renders null', () => {
+        const Component = ({projectData}) => <div>{projectData}</div>;
+        const WrappedComponent = HashParserHOC(ProjectLoaderHOC(Component));
+        window.location.hash = '#winning';
+        const originalLoad = storage.load;
+        storage.load = jest.fn(() => Promise.resolve(null));
+        const mounted = mount(<WrappedComponent />);
+        storage.load = originalLoad;
+        window.location.hash = '';
```

Comment:
This is how it was before, but it's kind of confusing how this works. The above two lines are just setting global state to how it was found, so the test is idempotent. Either we should do this at the end of all the tests, or we should leave a comment here that says that's what we're doing.